manifest.versions.json

```diff
@@ -41,6 +41,7 @@
       "lcu|ltsc2025|default": "$(lcu|ltsc2025|4.8.1)",
 
       "nuget|version": "6.12.2",
+      "referenceassemblies|version": "1.0.3",
```

Comment:
We should have an issue logged for keeping this updated. I understand it rarely updates, but still.

Reply:
If/when this is merged, I will add it to the servicing checklist just like we check for NuGet CLI updates.